### DISCUSSION TITLE
求暗黑血统一代

### DISCUSSION THREAD

**Discussion by 44654681** [Gamuxorg/bbs#412] [投递和请求游戏]
我看了一下，这个游戏一代有linux版本的，但是我找不到。

**Comment by hyh19962008** [comment]
steam上显示不支持Linux呢
________________________________
发件人: 44654681 ***@***.***>
发送时间: 2023年5月31日 9:22
收件人: Gamuxorg/bbs ***@***.***>
抄送: Subscribed ***@***.***>
主题: [Gamuxorg/bbs] 求暗黑血统一代 (Discussion #412)


我看了一下，这个游戏一代有linux版本的，但是我找不到。

―

**Reply by 44654681** [reply]
> steam上显示不支持Linux呢
> […](#)
> ________________________________ 发件人: 44654681 ***@***.***> 发送时间: 2023年5月31日 9:22 收件人: Gamuxorg/bbs ***@***.***> 抄送: Subscribed ***@***.***> 主题: [Gamuxorg/bbs] 求暗黑血统一代 (Discussion #412) 我看了一下，这个游戏一代有linux版本的，但是我找不到。 ― Reply to this email directly, view it on GitHub<#412>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFRK4UPYLVGYFOZLE7CJ7WTXI2MNFANCNFSM6AAAAAAYUX74XY>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

那就wine吧
